{"level": "ERROR", "timestamp": "2024-03-04T11:55:23.004238", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T11:55:41.617190", "message": "Error getting comment details: 'NoneType' object is not iterable", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 35, "function": "get_details"}
{"level": "ERROR", "timestamp": "2024-03-04T11:55:47.006284", "message": "Error inserting comment: ", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 50, "function": "insert_obj"}
{"level": "ERROR", "timestamp": "2024-03-04T11:55:58.674087", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:17:18.076612", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:19:24.738514", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:20:41.722068", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:24:29.215793", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:26:47.740871", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:28:29.512216", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:30:01.549567", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:31:09.972188", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:33:41.752519", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T12:47:24.225384", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:08:10.627277", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:09:35.482145", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:10:04.721143", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:11:21.210879", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:11:47.328452", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:11:48.240333", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:14:37.220341", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:16:02.403348", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:17:04.137644", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:17:06.049898", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:17:51.364820", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:17:52.125987", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:18:49.176608", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:18:59.098878", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 101, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T13:46:37.501119", "message": "Error getting comment details: 'NoneType' object is not iterable", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 36, "function": "get_details"}
{"level": "ERROR", "timestamp": "2024-03-04T13:46:39.639582", "message": "Error getting comment details: 'NoneType' object is not iterable", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 36, "function": "get_details"}
{"level": "ERROR", "timestamp": "2024-03-04T13:46:40.690852", "message": "Error getting comment details: 'NoneType' object is not iterable", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 36, "function": "get_details"}
{"level": "ERROR", "timestamp": "2024-03-04T13:46:41.564962", "message": "Error getting comment details: 'NoneType' object is not iterable", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 36, "function": "get_details"}
{"level": "ERROR", "timestamp": "2024-03-04T13:51:04.491150", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.184798", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.475176", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.475803", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.724546", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.725534", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.977761", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:00.978586", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:01.214175", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:01.215168", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:01.438938", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:01.439981", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:02.666389", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:02.667081", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.010380", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.011215", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.214105", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.216710", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.464665", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.465498", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.685715", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.686941", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.899787", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:03.900884", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:04.543390", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:04.544306", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:04.960068", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:04.961202", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.110299", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.111240", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.247262", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.249276", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.401724", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.403671", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.561150", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:05.562509", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.082921", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.084278", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.479849", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.480589", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.612360", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.613524", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.767575", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.769273", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.924752", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:06.926484", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.074151", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.075401", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.517581", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.518525", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.828306", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.829187", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.974885", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:07.976080", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.115226", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.116588", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.262341", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.263550", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.406020", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.407713", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.934710", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:08.935800", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.306838", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.307477", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.431128", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.432228", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.576709", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.578432", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.727801", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.729191", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.877317", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:09.879167", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.375212", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.375629", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.679976", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.680659", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.809833", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.810744", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.940719", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:10.941608", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.079733", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.080716", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.210237", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.211039", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.794650", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:11.795210", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.104289", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.104728", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.221497", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.222120", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.336596", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.337264", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.458777", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.459581", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.581506", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.582046", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.989559", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:12.990379", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.320728", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.321521", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.451811", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.452533", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.565443", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.565801", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.677587", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.678105", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.855688", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:13.856170", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.271801", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.272719", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.606273", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.606824", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.730502", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.731355", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.860957", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.861948", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.991103", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:14.992351", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.109063", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.109587", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.526557", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.527239", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.853285", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.853963", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.974673", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:15.975423", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.092709", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.093305", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.210961", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.211636", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.334812", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.335624", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.753272", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:16.753743", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.063073", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.063867", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.186862", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.187765", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.308086", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.308799", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.425345", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.425854", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.540144", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.540636", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.933473", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:17.934039", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.259454", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.260062", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.374923", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.375405", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.490965", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.491527", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.612680", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.613253", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.734589", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:18.735511", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.171706", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.172650", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.496759", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.497465", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.629584", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.630418", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.761862", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.762868", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.881534", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.882124", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.998886", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:19.999449", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.407913", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.408848", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.741727", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.742476", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.865157", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.866158", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.989825", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:20.990391", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.106759", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.107276", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.224089", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.224602", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.649103", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:21.650069", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.009597", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.010161", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.126246", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.126821", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.253413", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.254182", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.377127", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.377852", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.507543", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:23.508446", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.131501", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.132178", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.240653", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.241383", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.350141", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.350769", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.451240", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.451882", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.537957", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.538552", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.648134", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:28.648782", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.423056", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.423479", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.533890", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.534392", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.633528", "message": "Error getting user strenght: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 70, "function": "strenght"}
{"level": "ERROR", "timestamp": "2024-03-04T13:52:29.634194", "message": "Error getting user simple details: 'User' object has no attribute 'like_on'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/config/models.py", "line": 128, "function": "simple_dict"}
{"level": "ERROR", "timestamp": "2024-03-04T18:42:08.627202", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 102, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T18:49:11.951623", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 104, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T19:00:42.382817", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 104, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T19:01:51.373602", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 112, "function": "patchCommentConnections"}
{"level": "ERROR", "timestamp": "2024-03-04T19:02:51.854878", "message": "Error patching comment connections: add_user_to_post() got an unexpected keyword argument 'post_node'", "file": "/Users/victoralmeida/Dropbox (company01 Global)/Mac/Documents/company01_internal/dev_community/backend/dev_community-developement/models/handlers/comment.py", "line": 112, "function": "patchCommentConnections"}
